package.json

```diff
@@ -58,6 +58,7 @@
     "eslint-plugin-standard": "^5.0.0",
     "hardhat": "^2.11.2",
     "hardhat-contract-sizer": "^2.6.1",
+    "hardhat-tracer": "^2.0.0",
```

Comment:
Please extract this as a separate PR. I think you included this in another PR as well.